build/Dockerfile

```diff
@@ -61,6 +58,12 @@ ENV USER_NAME=psap-ci-runner \
 
 COPY . ${HOME}/
 
+# Prepare the CI `run` entrypoint
+RUN echo -e '#!/usr/bin/env bash \n\
```

Comment:
I guess it's too early to use https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/

Reply:
I tried the multiline ```
echo ' ...
...
...'
```
but it failed, not sure if this one has a chance to be supported ... ?